### PR TITLE
separated security norm/inform statements and updated references

### PR DESCRIPTION
**Pull request by 2byrds**

### CODE REVIEW

File: spec/appendix.md [hunk 1]
```diff
@@ -66,6 +69,9 @@ See also: decentralized identifier, [ref: self-certifying identifier (SCID)].
 [[def: key event log (KEL), KEL, KELs]]
 ~ A verifiable append-only log of [[ref: key events]] for an [[ref: AID]] that is both backward and forward-chained, as defined by the [[ref: KERI specification]]. See [WebOfTrust glossary](https://weboftrust.github.io/WOT-terms/docs/glossary/key-event-log) for more detail.
 
+[[def: KEL backed data]]
+~ [[ref: KEL]] backed data in `did:webs` provides the highest level of secure data and can either be found in the KEL or found anchored to the KEL. This means the signatures on the events in the KEL are strongly bound to the key state at the time the events are entered in the KEL. This provides the strongest guarantee of duplicity evidence so that any verifier is protected. The information is end-verifiable and any evidence of duplicity means do-not-trust. See [WebOfTrust glossary](https://weboftrust.github.io/WOT-terms/docs/glossary/kel-backed-data) for more detail.
```
Comment (daidoji) [inline spec/appendix.md:73]:
```suggestion
~ [[ref: KEL]] backed data in `did:webs` provides the highest level of data security assurance and such data can be found either in the KEL or anchored to an event in the KEL. This means that the signatures on the events in the KEL are strongly bound to the key state at the time the events are entered in the KEL, that is the data. This provides strong guarantees of non-duplicity to any verifiers receiving a presentation as the KELs are protected and can be watched by agents ([[ref: watcher]]) of the verifiers. The information is end-verifiable and any evidence of duplicity in the events is evidence that the data or presentation should not be trusted. See [WebOfTrust glossary](https://weboftrust.github.io/WOT-terms/docs/glossary/kel-backed-data) for more detail.
```

Comment (daidoji) [reply spec/appendix.md:73]: This def sounded a little strange so I reworded it a bit.  Feel free to take some or all of it (or none of it if you think the original was good).

File: spec/appendix.md [hunk 2]
```diff
@@ -81,6 +87,9 @@ See also: decentralized identifier, [ref: self-certifying identifier (SCID)].
 [[def: key state, key states]]
 ~ The set of currently authoritative key pairs (current keys) for an [[ref: AID]] and any other information necessary to secure or establish control authority over the AID. See [WebOfTrust glossary](https://weboftrust.github.io/WOT-terms/docs/glossary/key-state) for more detail.
 
+[[def: KERI Request Authentication Mechanism, KRAM]]
+~ A non-interactive replay attack protection algorithm that uses a sliding window of date-time stamps and key state (similar to the the tuple as in [[ref: BADA-RUN]]) but the date-time is the replier’s not the querier’s. **KRAM is meant to protect a host**. See the [WebOfTrust glossary](https://weboftrust.github.io/WOT-terms/docs/glossary/keri-request-authentication-method) for more detail.
```
Comment (daidoji) [inline spec/appendix.md:91]:
```suggestion
~ A non-interactive replay attack protection algorithm that uses a sliding window of date-time stamps and key state (similar to the tuple in [[ref: BADA-RUN]]) but the date-time is the replier’s not the querier’s. **KRAM is meant to protect a host**. See the [WebOfTrust glossary](https://weboftrust.github.io/WOT-terms/docs/glossary/keri-request-authentication-method) for more detail.
```

File: spec/security_considerations.md [hunk 3]
```diff
@@ -3,106 +3,75 @@ There are many security considerations related to web requests, storing informat
...
-- Denial of service (DoS) attack
-- Deletion attack
-- Duplicity detection
-- Eclipse attack
-- Forgery
-- Impersonation attack
-- Key Compromise attack
-- Malleability attack
-- Replay attack
+1. All `did:webs` features MUST narrow the attack surface against common threats, such as:
```
Comment (daidoji) [inline spec/security_considerations.md:6]: "narrow" => "reduce or protect"?

**Review by daidoji — COMMENTED** [review]
A few suggestions on wording but overall looks good.

**Review by daidoji — APPROVED** [review]

File: spec/security_considerations.md (hunk 3 above)
Comment (2byrds) [reply spec/security_considerations.md]: @daidoji thank you for review. I have accepted the suggestions and agree that 'reduce' is better and have changed that now.

**Review by daidoji — APPROVED** [review]

**Pull request merged**